build/tasks/build.js

```diff
@@ -82,13 +89,15 @@ module.exports = function( grunt ) {
 			contents = contents
 				.replace( /define\(\[[^\]]*\]\)[\W\n]+$/, "" );
 		}
+
 		// AMD Name
-		if ( (amdName = grunt.option( "amd" )) != null && /^exports\/amd$/.test( name ) ) {
-			if (amdName) {
+		if ( ( amdName = grunt.option( "amd" ) ) != null && /^exports\/amd$/.test( name ) ) {
```

Comment:
>
> I'm less bothered by the extra spaces in expression parenthesis as I am by
> the extra space at the end of our methods (i.e. } );), which I don't
> think would require changes to jscs if we were comfortable with things like method(
> arg1, { object: value }).
>

I like those spaces for consistency reasons. If the majority doesn't want
them, I'm fine with this exception (although I prefer not to have too many
exceptions as @mikesherov suggested) but it'd be good to have it checked or
we'll have half of the code in one style & half in the other.